content/zh-cn/docs/tasks/run-application/delete-stateful-set.md

```diff
@@ -111,9 +111,9 @@ Use caution when deleting a PVC, as it may lead to data loss.
 <!--
 ### Complete deletion of a StatefulSet
 
-To simply delete everything in a StatefulSet, including the associated pods, you can run a series of commands similar to the following:
+To delete everything in a StatefulSet, including the associated pods, you can run a series of commands similar to the following:
 -->
-### 完全删除 StatefulSet  {#complete-deletion-of-a-statefulset}
+### 完全删除 StatefulSet {#complete-deletion-of-a-statefulset}
```

Comment:
Changes like this are unnecessary.
Whitespaces are for readability. Unifying this into a single space doesn't add much value.